src/control/vendor/github.com/daos-stack/go-ipmctl/README.md

```diff
@@ -0,0 +1,41 @@
+# Go language bindings for the ipmctl API for managing PMMs
+
+Go bindings for the libipmctl native C lib to enable management of DCPMMs from Go source 
+
+This is a Go interface for [ipmctl](https://github.com/intel/ipmctl)
+
+## How to Build
+
+This is a [Go](https://golang.orghttps://golang.org/doc/install)
+project, so the Go development tools are naturally required. We
+recommend the most current Go release available. As of July 2018, the project has been built and tested with Go 1.11.5
```

Comment:
Do you want to update the date to at least June 2019?

Reply:
will do when I next  update go-ipmctl, that would require another PR on that reapo